From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Danila Tikhonov <danila@jiaxyga.com>
Date: Mon, 21 Apr 2025 21:59:45 +0300
Subject: [PATCH 018/166] FROMLIST: dt-bindings: clock: qcom,gpucc: Add the
 SM7150 compatible

SM7150 is fully compatible with the existing SC7180 GPU Clock
Controller driver. Define corresponding compatible string, having the
qcom,sc7180-gpucc as a fallback.

Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
---
 .../devicetree/bindings/clock/qcom,gpucc.yaml | 29 +++++++++++--------
 1 file changed, 17 insertions(+), 12 deletions(-)

diff --git a/Documentation/devicetree/bindings/clock/qcom,gpucc.yaml b/Documentation/devicetree/bindings/clock/qcom,gpucc.yaml
index 3ac4419009a9..516c702fdee7 100644
--- a/Documentation/devicetree/bindings/clock/qcom,gpucc.yaml
+++ b/Documentation/devicetree/bindings/clock/qcom,gpucc.yaml
@@ -28,18 +28,23 @@ description: |
 
 properties:
   compatible:
-    enum:
-      - qcom,qcs8300-gpucc
-      - qcom,sdm845-gpucc
-      - qcom,sa8775p-gpucc
-      - qcom,sc7180-gpucc
-      - qcom,sc7280-gpucc
-      - qcom,sc8180x-gpucc
-      - qcom,sc8280xp-gpucc
-      - qcom,sm6350-gpucc
-      - qcom,sm8150-gpucc
-      - qcom,sm8250-gpucc
-      - qcom,sm8350-gpucc
+    oneOf:
+      - enum:
+          - qcom,qcs8300-gpucc
+          - qcom,sdm845-gpucc
+          - qcom,sa8775p-gpucc
+          - qcom,sc7180-gpucc
+          - qcom,sc7280-gpucc
+          - qcom,sc8180x-gpucc
+          - qcom,sc8280xp-gpucc
+          - qcom,sm6350-gpucc
+          - qcom,sm8150-gpucc
+          - qcom,sm8250-gpucc
+          - qcom,sm8350-gpucc
+      - items:
+          - enum:
+              - qcom,sm7150-gpucc
+          - const: qcom,sc7180-gpucc
 
   clocks:
     items:
